weldx/welding/utility.py

```diff
@@ -35,7 +32,7 @@ def compute_welding_speed(
     """
     groove_area = groove.cross_sect_area
     wire_area = np.pi / 4 * wire_diameter ** 2
-    weld_speed = wire_area * wire_feed / groove_area * seam_length
```

Comment:
I have removed this, output should now be `[length]/[time]` @marscher

Reply:
Thanks. I should also add a small test for that.

Reply:
I feel like it should be safe the way that it is now.
The unit error would raise on the `wx_unit` validation here anyway https://github.com/BAMWelDX/weldx/blob/2bda4dd9e0b0a909a1ba6054a7e3cdd3d5cd9ea9/weldx/asdf/schemas/weldx.bam.de/weldx/datamodels/single_pass_weld-1.0.0.schema.yaml#L29-L34